XRTK.WebXR/Packages/manifest.json

```diff
@@ -16,7 +16,7 @@
     "com.unity.test-framework": "1.1.19",
     "com.unity.textmeshpro": "2.1.3",
     "com.unity.ugui": "1.0.0",
-    "com.xrtk.core": "0.2.0",
+    "com.xrtk.core": "0.2.9",
```

Comment:
remove this entry here, since the dependency is defined in the `package.json`

Reply:
Updated in latest patches